src/model/user.rs

```diff
@@ -371,7 +371,7 @@ impl User {
     /// [`crate::http::Http::get_user`].
```

Comment:
This doc comment is wrong now, can you update it/remove the note?

Reply:
Donezo